src/components/GithubAccessBlocked.js

```diff
@@ -1,28 +1,35 @@
-import React from 'react'
+import React, {useEffect, useState} from 'react'
 import {
     Box,
     Grid,
     Typography,
     Link
 } from '@material-ui/core'
-import { useQuery } from '@apollo/client';
+import {useLazyQuery, useQuery} from '@apollo/client';
 import { GET_PROJECT } from '../operations/queries/ProjectQueries';
 
 const GithubAccessBlocked = (props) => {
 
     const githubURL = props.githubURL
-
-    const {
+    
+    const [project, setProject] = useState(null)
+    
+    const [getProject, {
         data: dataProject,
         loading: loadingProject,
         error: errorProject,
-    } = useQuery(GET_PROJECT, {
-        variables: {
-            id: Number(props.projectId)
+    }] = useLazyQuery(GET_PROJECT, {
+        onCompleted: dataProject => {
+            setProject(dataProject.getProjectById.github_url)
```

Comment:
Two things can be done here,

1. set the `project` state value to the whole `dataProject.getProjectById` object and on the Link href set to`project.github_url`
2. change the state variable name to `projectUrl`. Is a little confusing having a variable called project that only stores the project url and not the complete project object